src/devices/xyzroe.ts

```diff
@@ -463,6 +467,30 @@ const definitions: DefinitionWithExtend[] = [
             ]);
         },
     },
+    {
+        zigbeeModel: ['ZigUSB_C6'],
+        model: 'ZigUSB_C6',
+        vendor: 'xyzroe',
+        description: 'Zigbee USB switch with monitoring',
+        extend: [
+            deviceEndpoints({endpoints: {1: 1, 2: 2, 3: 3}}),
+            identify(),
+            electricityMeter({
+                cluster: 'electrical',
+                attributes: 'both',
+                configureReporting: electroReporting,
+                endpointNames: ['1'],
+            }),
+            temperature({
+                reporting: tempReporting,
```

Comment:
Why do we need custom reporting values here? (same for `electroReporting`)

Reply:
Because as a device developer I think it is more correct to use it with such settings. I perfectly understand and agree with your comments about the general code, but I can’t understand why the device should use the standard one??

Reply:
>  but I can’t understand why the device should use the standard one??

Because it's nice to have unification, why should 1 plug report every 1w and others every 10w? We should provide decent defaults, users can always configure it differently from the frontend (in case of a specific usecase)

Reply:
But devices are different, with different measurement limits. In this case, the device measures the consumption of a USB device at 5V, and there cannot be values greater than 10 W in principle.

Reply:
That justifies custom reporting values for the `electricityMeter` indeed!

What about the ones for temperature?

Reply:
I assumed that this parameter is insignificant in this device, since it essentially displays the temperature of the device, and in order to reduce the load on the network, I decided to reduce reporting relative to the standard values.
But if unification is more important, of course I can return the standard ones.

Reply:
Let's return to the standard ones then